lib/matplotlib/backends/backend_pdf.py

```diff
@@ -698,9 +736,9 @@ def _write_afm_font(self, filename):
         self.writeObject(fontdictObject, fontdict)
         return fontdictObject
 
-    def embedTeXFont(self, texname, fontinfo):
```

Comment:
I don't think that's something that an external user would ever have called. Let's mark the function private with an underscore and document the change.